spec/controllers/key_registration.rb

```diff
@@ -0,0 +1,62 @@
+require 'spec_helper'
```

Comment:
It _looks_ like it hasn't run as a result of the naming, as the code coverage indicates that the `KeyRegistration` class hasn't been hit.